docs/controls/wpf-winforms/WebView-known-issues.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Known Issues for WebView control for Windows Forms and WPF
+title: Known Issues in WebView control

```

Comment:
Still poor grammar in this title.

also

Why remove important distinguishing information (that this applies to WinForms & WPF only) from the title? Surely the inclusion of product names is good for clarity (and SEO)